content/hi/training/_index.html

```diff
@@ -0,0 +1,154 @@
+---
+title: प्रशिक्षण
+bigheader: कुबेरनेट्स प्रशिक्षण और प्रमाणन
+abstract: प्रशिक्षण कार्यक्रम, प्रमाणपत्र, और भागीदार।
+layout: basic
+cid: training
+class: training
+---
+
+<section class="call-to-action">
+  <div class="main-section">
+    <div class="call-to-action" id="cta-certification">
+      <div class="cta-text">
+        <h2>अपना क्लाउड नेटिव करियर बनाएं</h2>
+        <p>कुबेरनेट्स क्लाउड नेटिव आंदोलन के मूल में है। लिनक्स फाउंडेशन और हमारे प्रशिक्षण भागीदारों से प्रशिक्षण और प्रमाणन आपको अपने करियर में निवेश करने, कुबेरनेट्स सीखने और अपनी क्लाउड नेटिव परियोजनाओं को सफल बनाने की सुविधा देता है।</p>
```

Comment:
'आंदोलन' is not an appropriate word to use here
```suggestion
        <p>कुबेरनेट्स क्लाउड नेटिव आंदोलन के मूल में है। लिनक्स फाउंडेशन और हमारे प्रशिक्षण कंट्रीब्यूटरों से प्रशिक्षण और प्रमाणन आपको अपने करियर में निवेश करने, कुबेरनेट्स सीखने और अपनी क्लाउड नेटिव प्रोजेक्ट्स को सफल बनाने की सुविधा देता है।</p>
```